src/System.Threading.Overlapped/tests/ThreadPoolBoundHandle_BindHandleTests.cs

```diff
@@ -10,6 +10,7 @@
 public partial class ThreadPoolBoundHandleTests
 {
     [Fact]
+    [PlatformSpecific(TestPlatforms.Windows)] // ThreadPoolBoundHandle.BindHandle is not supported on Unix
```

Comment:
Per my comments in the CoreCLR PR, I'm wondering if this test should actually pass on both platforms.  Generally we still do arg validation the same across platforms even if the implementation on one of them will then throw a PNSE.